synapseclient/core/upload/multipart_upload_async.py

```diff
@@ -684,15 +685,16 @@ def part_fn(part_number: int) -> bytes:
         """Return the nth chunk of a file."""
         return get_file_chunk(file_path, part_number, part_size)
 
-    return await _multipart_upload_async(
-        syn,
-        dest_file_name,
-        upload_request,
-        part_fn,
-        md5_fn_util,
-        force_restart=force_restart,
-        storage_str=storage_str,
-    )
+    with logging_redirect_tqdm(loggers=[syn.logger]):
+        return await _multipart_upload_async(
+            syn,
+            dest_file_name,
+            upload_request,
+            part_fn,
+            md5_fn_util,
+            force_restart=force_restart,
+            storage_str=storage_str,
+        )
```

Comment:
I was noticing some weirdness when I had multiple progress bar running (One for the chunked upload of row data into a Table, and one for the actual upload of the file itself).

This doesn't totally fix it, but the behavior seems better with it like this.

Reply:
Great find. This is likely the issue we're encountering here. I will create a tech debt ticket to go back and look at these.

Reply:
https://sagebionetworks.jira.com/browse/SYNPY-1488